Formula/plplot.rb

```diff
@@ -11,6 +11,11 @@ class Plplot < Formula
     sha256 "b779762659e485d6c9cad54206b1e72f2db5e82950b19a356439e9ce3ef79138" => :mountain_lion
   end
 
+  patch do
```

Comment:
Need a comment with links to upstream patches (the SF commits should do) and brief explanation of what the patch does.